### ISSUE TITLE
Sharing link not working

### ISSUE THREAD

**Issue by FromScratchCommunications** [baggachipz/tinylist#58]
I'm trying to share the following list – but it turns up empty: https://tinylist.app/share/ts3ba2fb65-cb9f-40d0-ac30-774f090e58e6
The link was created by pressing the Share dialogue symbol on the checklist itself, then confirming and either choosing Copy to Clipboard or Email (I tried both and neither is working).

**Comment by baggachipz** [comment]
Did you ever have this work? I just verified from my phone to my laptop and it did work. Sometimes you have to give it a couple minutes to start up and sync; if that doesn't work, try reloading the page and giving it a minute.

**Comment by baggachipz** [comment]
I just tried yours and see it didn't work. I'll look into this soon.

**Comment by ItsCheeseGromit** [comment]
Same for me.
And as a matter of fact, when I click on OP's link, it opens my own list.

@baggachipz are you still working on this app? I hope so - I just found it and it looks like exactly what I need!

**Comment by baggachipz** [comment]
Yeah, I'm still here. I've had a tough 2 years with health and some other things. I will fix this in the near future, I want this thing to be great. I use it everyday but not the sharing feature, I admit.

**Comment by ItsCheeseGromit** [comment]
I'm sorry to hear you've had troubles, hopefully things are looking up for you.

Glad to hear you're still working on the project, and thank you. I tried to buy you a coffee, but I dont see any links to do so.
